<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorConfigPage</class>
 <widget class="QWidget" name="EditorConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Indentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="topMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="spaceIndent">
         <property name="text">
          <string>Use spaces instead of tabs</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Indentation width:</string>
           </property>
           <property name="indent">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="indentWidth">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="suffix">
            <string> spaces</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Code Evaluation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="stepForwardEvaluation">
         <property name="text">
          <string>Step forward after evaluating a line</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Normal Text</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Matching Brackets</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ScIDE::ColorWidget" name="bgColor"/>
       </item>
       <item row="1" column="1">
        <widget class="ScIDE::ColorWidget" name="textColor"/>
       </item>
       <item row="2" column="1">
        <widget class="ScIDE::ColorWidget" name="bracketColor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Syntax Highlighting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ScIDE::TextFormatListWidget" name="textFormatList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScIDE::TextFormatListWidget</class>
   <extends>QTreeView</extends>
   <header>text_format_list_widget.hpp</header>
  </customwidget>
  <customwidget>
   <class>ScIDE::ColorWidget</class>
   <extends>QLabel</extends>
   <header>color_widget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
